veps/sig-network/passt-migration-proposal.md

```diff
@@ -0,0 +1,333 @@
+- Support for observability tools and service mesh technologies.
+- Enhanced IPv6 support: Supporting dynamic configuration for the guests (RA, DHCPv6).
+- High network throughput, on par with the masquerade network binding.
+
+## Non Goals
+
+- Supporting network protocols other those that are [currently supported](https://passt.top/passt/about/#protocols).
+- Supporting CNIs that do not provide sticky IP addresses during migration (migration succeeds, but existing connections break).
+- Deployment of passt binding plugin.
+
+## Definition of Users
+
+- Cluster Admins
+- VM/Namespace owners
+- Guest VM users – typically unaware of their network binding
+
+## User Stories
+
+- As a Cluster Admin and as a Namespace Owner, I require seamless VM migration to perform infrastructure upgrades without affecting business continuity, preserving connectivity even for sensitive applications and protocols.
+- As a Guest VM user, I want optimal network performance comparable to masquerade, ensuring my virtualized applications perform well.  
+- As a Cluster Admin and as a Namespace Owner, I want observability and service meshes for enhanced monitoring, encryption, and load balancing.
+- As a Cluster Admin, I want to make sure that I can upgrade existing VMs to the new binding and that the procedure to do so is clear.
+
+## Repos
+
+- [https://github.com/kubevirt/kubevirt](https://github.com/kubevirt/kubevirt)
+
+## Design
+
+The migration flow consists of many moving parts, some of which are considered external to KubeVirt or dependencies of the solution:
+- `passt` and `passt-repair` perform the technical handling of the network sockets, and communicate with each other.
+- `libvirt` and `QEMU` are responsible to invoke `passt`, communicate with it and carry out the actual migration.
+
+As this proposal focuses on integration to KubeVirt, it covers the following parts:
+- The DomainXML format will to be adapted to support `vhost-user` transport.
+- The image build process of virt-handler will be enhanced to include the `passt-repair` binary.
+- `virt-handler` will be enhanced to invoke `passt-repair` for migration source and target in precise timing.
+- The build and deployment processes of the `passt` network binding plugin are in incomplete state and therefore require completion. 
+
+### About passt
+
+`passt` is a lightweight user-space networking solution designed for virtualized environments. It allows guest VMs to access the host’s network stack without requiring root privileges or additional network bridges. 
+It is already used in KubeVirt as a network binding plugin, shipped with the virt-launcher container image, and is documented in the [user guide](https://kubevirt.io/user-guide/network/net_binding_plugins/passt/#passt-binding).
+In [release 2025_02_17.a1e48a0](https://passt.top/passt/tag/?h=2025_02_17.a1e48a0) it has been enhanced to communicate with the guest directly using `vhost-user` transport.
```

Comment:
This seems to repeat the overview. If there are part that do not repeat, please move it there. It does not have anything to do with the solution, I think. You could ref to that overview section from the previous section.